_posts/2018-09-27-SpringOne-Platform.md

```diff
@@ -0,0 +1,151 @@
+Because there were A LOT of sessions, I wasn't able to gather or note down everything.
+All sessions except for the workshops were recorded and I'll update this post to get you directed straight to the videos on YouTube when they come online!
+
+## Announcements
+
+I listed some of the major announcements that were made during SpringOne.
+You can find a link to the corresponding video at the end of each topic.
+
+### AWS Service broker for PCF, open beta
+
+Amazon will bring the AWS Service Broker to PCF, making it easier for you to connect your applications to its service catalog of over 18 services.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=8m42s" target="_blank">here</a>.
+
+### Pivotal Function Service
+
+Next to PAS (Pivotal Application Service) and PKS (Pivotal Container Service), Pivotal will expand PCF with PFS (Pivotal Function Service) which will be build on top of Knative.
+Pivotal Greenplum, which is its "Postgres for Petabytes", will also be coming to PCF in the near future so that the same easy-to-use experience that developers nowadays already have with the platform will be extended to the data scientists and data engineers.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=9m34s" target="_blank">here</a>.
+
+### First-timers-only issues
+
+Because it can be daunting for people to enter a community they are creating "first-timers-only" issues on the Spring Boot project, making it easier for newcomers to get connected.
+While working on these issues developers of the Spring Boot project will come alongside you to teach you to contribute back, help you and coach you.
+This way you will get to become part of the Spring community.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=10m39s" target="_blank">here</a>.
+
+### Pivotal Tracker: Maestro
+
+> What happens when your organization scales, and you end up with lots of fast-moving developers?
+> It's really, really easy to get misaligned.
+> And when you get misaligned, it's really easy to ship the wrong thing.
+
+That's why Pivotal has build a tool to help teams articulate and align around business outcomes to give organizations the high-level view so they can assure that everyone is aligned and delivering value.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=11m43s" target="_blank">here</a>.
+
+### Pivotal Act
+
+> "Pivotal Act is a program that partners with humanitarian organizations and charities to identify, design, and develop practical solutions to pressing challenges around the world."
+
+Together with developers, designers, and engineers from Pivotal Labs, Pivotal is going to apply the same methodology like they use in other engagements with clients but tailored to the needs of the humanitarian and social impact sectors instead of just donating technology or funds.
```

Comment:
Pivotal is going to apply the same methodology ~~they~~ use in other engagements with clients **,** but tailored